openhtf/plugs/uart.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
I agree with you for a standalone file that the file name should match what's defined inside.

That said our motivation for the current naming convention of uart/serialplug is to match the existing usb/adbplug such that the file or folder name is the physical interface used (uart and usb) and the plug name specifies the protocol (serial and adb).  This way we can add additional uart based protocol later if necessary.

What do you think?